docs/investigations/secure-auth-resolvers.md

```diff
@@ -0,0 +1,105 @@
+# Limit access of GraphQL objects or fields for specific resource owners
+
+### Case 1:
+
+Every object should has access to read information only about itself.
+We should restrict access to other objects of the same type, for example: runtime musn't read other runtime configuraion.
+We have two kind of such resolvers:
+* `runtime(ID)` runtime should be able to only read about itself
+* `runtimes` runtime should get list only with itself
+
+Also we have fields with custom resolvers like this:
+In `APIDefinition`, there is a field `auth` with param `runtimeID`, runtime which ask for 
+this field should be validated if ask with own `ID`.
+
+
+#### Example:
```

Comment:
This example somehow repeats previous section, maybe we should join them and get rid of  Example section